MycroftAI/mycroft-core
# add missing pt-pt dialog files

## Description
New dialog resource files have been added, this PR adds the missing dialogs for pt-pt

## How to test
use google translate and see if it makes sense to you, ask a native portuguese speaker to nitpick

## Contributor license agreement signed?
CLA [x ] (Whether you have signed a [CLA - Contributor Licensing Agreement](https://mycroft.ai/cla/)
